src/pages/Stores/components/BlockStore/styles.scss

```diff
@@ -0,0 +1,26 @@
+.block-store {
```

Comment:
I'd say it makes more sense it being presented as a table, you could reuse the table used for Members